gaupol/agents/video.py

```diff
@@ -348,11 +352,17 @@ def _update_languages_menu(self):
         """Update the audio language selection menu."""
         menu = self.get_menubar_section("audio-languages-placeholder")
         menu.remove_all()
-        languages = self.player.get_audio_languages()
-        for i, language in enumerate(languages):
-            language = language or _("Undefined")
+        tracks = self.player.get_audio_infos()
+        for i, track in enumerate(tracks):
+            title = track.title or "{} {}".format(_("Track"), i + 1)
```

Comment:
The part on the right would be i18n-wise better as `_("Track {:d}").format(i + 1)`. The intent would be clearer and also if some weird language wants to flip the number first, that would be doable.